src/components/Api.js

```diff
@@ -0,0 +1,80 @@
+export default class Api {
+  constructor(options) {
+    this._url = options.url;
+    this._headers = options.headers;
+  }
+
+  _getRes(res) {
```

Comment:
Great that you've implemented a  method for processing server response, so the code that checks the server response and converts it from JSON  is reused